src/index.js

```diff
@@ -1 +1,21 @@
-import './style.css';
\ No newline at end of file
+import { addScore } from './modules/addscore.js';
+import getscores from './modules/getscore.js';
+import './style.css';
+
+const form = document.querySelector('.form');
+const refreshBtn = document.querySelector('.refresh');
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const nameInput = document.querySelector('.name').value;
+  const scoreInput = document.querySelector('.score').value;
+  addScore(nameInput, scoreInput);
+  document.querySelector('.name').value = '';
+  document.querySelector('.score').value = '';
```

Comment:
[OPTIONAL] It would be a good idea that the name and score fields should be properly validated to ensure a user cannot submit blank entries. :nerd_face: :nerd_face: